webClient/src/app/app.component.ts

```diff
@@ -302,6 +304,19 @@ export class AppComponent implements AfterViewInit {
     }
   }
 
+  checkZssProxy(): Promise<any> {
+    return new Promise((resolve, reject) => {
+      if (this.host === "") {
+        this.loadZssSettings().subscribe((zssSettings: ZssConfig) => {
```

Comment:
This works properly when ZSS exists. The server can technically run without ZSS, so in the case that this request fails, or zssSettings.zssServerHostName doesn't exist, you need an answer. Currently what it does if /server/proxies doesn't exist (throws a 404) or has different values than expected is that this hangs - terminal shows no error but does nothing.

I think the best thing to do is sanity check followed by use of `this.setError`